Snowflake.Data.Tests/Client/Zip.cs

```diff
@@ -0,0 +1,40 @@
+namespace Snowflake.Data.Tests.Client
+{
+    public class Zip
```

Comment:
if those test classes refer to structure types only could we at least put them into a dedicated folder within the tests suite?

Reply:
My bad, thanks!